README.md

```diff
@@ -30,13 +32,24 @@ Click on the "Use this template" button on the GitHub template repository page t
      ```bash
      cp .env.example .env
      ```
-   - Open the `.env` file.
+   - Open the new created `.env` file.
```

Comment:
**Line 15:** Corrected version: **Navigate to GitHub's token settings**:

**Line 18:** Token Settings - both capital

**Line 26:** "Set up" instead of "Setup"

This line:

```suggestion
   - Open the newly created `.env` file.
```